src/Symfony/Component/HttpKernel/Tests/HttpCache/EsiTest.php

```diff
@@ -92,6 +92,17 @@ public function testProcessDoesNothingIfContentTypeIsNotHtml()
         $this->assertFalse($response->headers->has('x-body-eval'));
     }
 
+    public function testRemoveEsiRemoveBlocks()
```

Comment:
`testMultilineEsiRemoveTagsAreRemoved()`?

By the way, what about `<esi:comment>`?